protocol-units/bridge/move-modules/sources/atomic_bridge_initiator.move

```diff
@@ -0,0 +1,527 @@
+module atomic_bridge::atomic_bridge_initiator {
+    use aptos_framework::event::{Self, EventHandle};
+    use aptos_framework::account::{Self, Account};
+    use aptos_framework::timestamp;
+    use aptos_framework::primary_fungible_store;
+    use aptos_framework::dispatchable_fungible_asset;
+    use aptos_std::aptos_hash;
+    use std::signer;
+    use std::vector;
+    use std::bcs;
+    use moveth::moveth;
+
+    // Constants to represent the state of a bridge transfer
+    const INITIALIZED: u8 = 0;
+    const COMPLETED: u8 = 1;
+    const REFUNDED: u8 = 2;
+
+    // Constants for error handling
+
+    const EINSUFFICIENT_AMOUNT: u64 = 0;
+    const EINSUFFICIENT_BALANCE: u64 = 1;
+    const EDOES_NOT_EXIST: u64 = 2;
+    const EWRONG_PREIMAGE: u64 = 3;
+    const ENOT_INITIALIZED: u64 = 4;
+    
+
+    // A struct to hold the details of a bridge transfer
+    struct BridgeTransfer has key, store {
```

Comment:
```suggestion
    struct BridgeTransferDetails has key, store {
```
And then you can remove the comment, the code becomes more self documenting.

Reply:
For this change, I think it's implied that the details are in the struct fields. So removing the comment is fine but we don't want to change the name of the struct to append `Details` at the end of the name. 

In the `atomic_bridge_counterparty` Move module it's named `BridgeTransfer`

Likewise it's named `BridgeTransfer` in [AtomicBridgeInitiator.sol](https://github.com/movementlabsxyz/movement/blob/main/protocol-units/bridge/contracts/src/AtomicBridgeInitator.sol) 

For consistency to keep the name of the transfer struct the same across contracts and modules seems to me that `BridgeTransfer` is good to stay as it is.

Reply:
Ah yes true, in the rust service its `BridgeTransferDetails`, all good lets leave it as it is.